.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: rust
 sudo: false
-rust:
-  - nightly
-env:
-  - FEATURES=""
-  - FEATURES="unstable"
+matrix:
+  include:
+    - rust: nightly
+    - rust: nightly
+      env:
+      - FEATURES="unstable"
 script: |
-  cargo build --verbose --features "${FEATURES}"
+  cargo build --verbose --features "${FEATURES}" &&
```

Comment:
Aha. It should work well, it won't pass tests if it doesn't build.  My own project failed to chain together test && bench, which actually obscured errors.